includes/admin/class-wc-rest-payments-promotions-controller.php

```diff
@@ -38,11 +38,12 @@ class WC_REST_Payments_Promotions_Controller extends WC_Payments_REST_Controller
 	const CACHE_DURATION = 300;
 
 	/**
-	 * Option key for dismissed promotions.
+	 * Option key for promotion dismissals.
+	 * Stores array of [promo_id => [variation_id => timestamp]].
 	 *
 	 * @var string
 	 */
-	const DISMISSED_PROMOTIONS_OPTION = '_wcpay_dismissed_promotions';
+	const PROMOTION_DISMISSALS_OPTION = '_wcpay_promotion_dismissals';
```

Comment:
Would this change affect any existing promotions that have already been dismissed?